examples/solids/elasticity.c

```diff
@@ -92,27 +90,11 @@ int main(int argc, char **argv) {
   // -- Set mesh file, polynomial degree, problem type
   ierr = PetscCalloc1(1, &app_ctx); CHKERRQ(ierr);
   ierr = ProcessCommandLineOptions(comm, app_ctx); CHKERRQ(ierr);
+  ierr = PetscCalloc1(1, &problem_functions); CHKERRQ(ierr);
+  ierr = RegisterProblems(problem_functions); CHKERRQ(ierr);
   num_levels = app_ctx->num_levels;
   fine_level = num_levels - 1;
 
-  if (app_ctx->problem_choice != ELAS_FSInitial_MR1) {
-    // -- Set Poison's ratio, Young's Modulus
-    ierr = PetscMalloc1(1, &units); CHKERRQ(ierr);
-    ierr = PetscMalloc1(1, &phys); CHKERRQ(ierr);
-    ierr = ProcessPhysics_General(comm, app_ctx, phys, phys_MR, units);
-    CHKERRQ(ierr);
-    if (fabs(app_ctx->nu_smoother) > 1E-14) {
```

Comment:
Ahha, this is the one dangling piece. We didn't really do a lot with this separate smoother physics object. Is this something that we want? Is there a useful analog of this for Mooney-Rivlin?

Reply:
We expressed Mooney-Rivlin with a Poisson ratio nu. (We were initially using bulk modulus, but Poisson ratio is easier for me to reason about.)

Reply:
Ah, cool. I went ahead and added another function list for updating smoother physics. This PR should be good to merge now.